src/System.IO.FileSystem.AccessControl/src/System/Security/AccessControl/FileSystemSecurity.cs

```diff
@@ -0,0 +1,291 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Security.Principal;
+using Microsoft.Win32.SafeHandles;
+
+namespace System.Security.AccessControl
+{
+    public abstract class FileSystemSecurity : NativeObjectSecurity
+    {
+        private const ResourceType s_ResourceType = ResourceType.FileObject;
+
+        internal FileSystemSecurity(bool isContainer)
+            : base(isContainer, s_ResourceType, _HandleErrorCode, isContainer)
+        {
+        }
+
+        internal FileSystemSecurity(bool isContainer, string name, AccessControlSections includeSections, bool isDirectory)
```

Comment:
Since this is internal, could you rename the `string name` parameter  to `string fileName` or `string path`? I don't think `name` clarifies that we are dealing with a file system location here.

Reply:
I agree with you but I want to avoid making changes in this mostly structural PR.